e2e/react-core/src/react-module-federation.test.ts

```diff
@@ -282,6 +282,63 @@ describe('React Module Federation', () => {
     }
   }, 500_000);
 
+  it('should support host and remote with library type var', async () => {
```

Comment:
Can this test run serve and check that we can navigate correctly to the remote from the host? 

Building should be fine regardless, but we'd get more value from the test if we can ensure it will navigate correctly

Reply:
yea I will update the e2e to include navigation